### PR TITLE
Add details on API response changes to CHANGELOG

### LINKED ISSUE #140: Add details about the API response format changes to CHANGELOG
Especially about how the property field is structured.

### PR DESCRIPTION
**Pull request by matthiasschaub**
### Corresponding issue
Closes #140

### CODE REVIEW

**Review by joker234 — COMMENTED** [review]
Some remarks:
* I would keep the lines you deleted.
* I wouldn't put the “How to update” section in between other changes, because it might get lost.
* I would use [`text`](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L6033) or something else as code type if you don't write valid JSON. Otherwise you'll get ugly rendering (compare here: https://github.com/GIScience/ohsome-quality-analyst/blob/update-changelog-api-response/CHANGELOG.md).
* If you want to include an exemplary response, you should state what kind of response it is (iterator, report, …). I would tend to not include the whole response schema, but put that somewhere in the documentation and state only what you have to change and link the exemplary responses in the docs.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,9 +4,20 @@
...
+
+Now the API response is a valid GeoJSON.
+The `geometry` and `properties` of the input GeoJSON Feature are preserved.
+The data and results of an indicator or of a report and its associated indicators
+are written in a flat hierarchy to the properties field of the GeoJSON.
+
+API response equates to the GeoJSON file output of CLI plus further metadata about the API.
+
+For example responses have a look at the [API documentation](/docs/api.md).
+
```
Comment (joker234) [inline CHANGELOG.md:20]:
```suggestion
- API response is a valid GeoJSON and equates to the CLI file output ([#100], [#130])
    - API output schema changes completely to a GeoJSON output
    - indicator and report data and results are written in a flat hierarchy to the properties field of the GeoJSON
    - Example responses can be found in the [API documentation](/docs/api.md)

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -48,7 +59,6 @@
 [#134]: https://github.com/GIScience/ohsome-quality-analyst/pull/134
 [#136]: https://github.com/GIScience/ohsome-quality-analyst/pull/136
 
-
```
Comment (joker234) [inline CHANGELOG.md:51]: ?

File: docs/api.md [hunk 3]
```diff
@@ -1,84 +1,94 @@
 # API
 
-
-## Response Schema
+## Response Examples
 
 Indicator:
```
Comment (joker234) [inline docs/api.md:5]:
```suggestion
### Indicator
```

File: docs/api.md [hunk 4]
```diff
@@ -1,84 +1,94 @@
...
+    "data.area": "...",
+    "data.pop_count_per_sqkm": "...",
+    "data.feature_count": "...",
+    "data.feature_count_per_sqkm": "..."
+  }
 }
 ```
 
 
 Report:
```
Comment (joker234) [inline docs/api.md:41]:
```suggestion
### Report
```

**Review by joker234 — COMMENTED** [review]
General question about the examples: why did you replace that many properties by `...`? I would only do that for (parts of) the larger/multi-line properties.

**Comment by joker234** [comment]
Ah and maybe you can update the API version in the examples to `0.5.0` already.

**Comment by matthiasschaub** [comment]
> General question about the examples: why did you replace that many properties by `...`? I would only do that for (parts of) the larger/multi-line properties.

I did this for large string values and for null values (See issue #145).

**Review by joker234 — APPROVED** [review]

**Pull request merged**